docs/reference/ccr/index-following.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[testenv="platinum"]
+[[index-following]]
+=== Index following
+
+Index following is the core feature provided by {ccr}. You can use the
+{ref}/ccr-put-follow.html[create follower API] to create a follower index and
+initiate the following of a leader index on a remote cluster.
+
+=== Initiating following
+
+When you submit a {ref}/ccr-put-follow.html[create follower request], the follower
+index is created on the follower cluster. Once the follower index is created,
+the <<remote-recovery, remote recovery>> process bulk transfers all of the Lucene
+segment files from the leader cluster to the follower cluster. By default this
+recovery process will be asynchronous in relationship to the
+{ref}/ccr-put-follow.html[create follower request]. The request will return prior
+to the <<remote-recovery, remote recovery>> process being completed. If you
+would like to wait on the process to complete, you can use the
+`wait_for_active_shards` parameter to do so.
+
+=== Index following operation
+
+Once the follower index is created, <<remote-recovery, remote recovery>> is
+complete, and the shards are active, the normal following process begins. During
```

Comment:
This overlaps some with the ccr-overview docs, perhaps link there?

I also wonder if these two docs (the new one being added here, and the overview doc) should be merged? I am asking this question out loud, I do not have a strong viewpoint one way or the other.